js/adopters_test.js

```diff
@@ -0,0 +1,83 @@
+const eclipsefdn_adopters_element = document.querySelectorAll(".eclipsefdn-adopters")[0];
+
+function testAdopters(working_group) {
+	fetch("/adopters/assets/js/adopters.json")
+	.then(response => {
+		if (response.ok) {
+			return response.json();
+		} else {
+			return Promise.reject(response);
+		}
+	})
+	.then(data => createWGAdopters(data.adopters, working_group))
+	.catch(error => console.log(error))
+}
+
+function createWGAdopters(data, working_group) {
+
+    let project_set = new Set();
+    for(let i=0; i<data.length; i++) {
+
+        // Loop the projects of this adopter
+        for(let j=0; j<data[i].projects.length; j++) {
+			let project = data[i].projects[j];
+
+            // If not exists this project
+            if( !project_set.has(project) ) {
+                project_set.add(project);
+                // add the title
+				createProjectHeader(project);
+			}
+			createAdopters(project, data[i]);
+		}
+	}
+}
+
+function createProjectHeader(project) {
+	const h2 = document.createElement('h2');
+	h2.textContent = project;
+	h2.setAttribute("id", project);
+	eclipsefdn_adopters_element.append(h2);
+	// add the button
+	const headerAnchor = document.createElement('a');
```

Comment:
We don't need const here, we can use var (const is more useful for things like default property values and the like)